developer/installation/installation-linux.md

```diff
@@ -0,0 +1,91 @@
+# Installation for Linux
+
+
+## Install prerequisites:
+
+### Install Node
+
+Download and run the installer from the [NodeJs site](https://nodejs.org)
+
+_(if you want to install via a package manager follow the instructions for your distro [here](https://nodejs.org/en/download/package-manager/))_
+
+
+### Install Build Tools and Package Requirements
+
+For Ubuntu/Debian
+
+```sh
+apt-get update
+
+apt-get install -y --no-install-recommends build-essential bzip2 curl ca-certificates git graphicsmagick python
+```
+
+For CentOS/RHEL
+
+```sh
+yum groupinstall "Development Tools"
+# add "Extra Packages for Enterprise Linux" repository for GraphicsMagick
+yum install epel-release  GraphicsMagick
+```
+
+### Install Meteor
+
+```sh
+curl https://install.meteor.com/ | sh
+```
+
+
+## Install Reaction
+
+### Install the Reaction command-line interface (CLI)
+
+```sh
+# install CLI
+npm install -g reaction-cli
+```
+
+### Create your first Reaction project
+
+```sh
+# clone Reaction, install NPM dependencies
+reaction init
+# change directory into new Reaction project
+cd reaction
+```
+
+If you don't want Reaction to be installed in the default `reaction` directory you can
+specify a direction like
+```sh
+reaction init my-new-reaction-project
+```
+
+
+### Start Reaction
+
+To start Reaction, run the `reaction` command
+
+```sh
+# start Reaction
+reaction
+# or
+reaction run
+```
+
+**Note that the first run can take a while as it downloads dependencies. This is especially true if you are not in North America**
```

Comment:
This sounds a bit like a joke, but probably is founded. Why do you think it's slower outside the U.S.? Because of lower bandwidth? Probably most people just don't know how fast your backbones really are. At least I don't know. I've heard of very fast connections in the bay area though.

Reply:
MDG just does not have any mirror servers in Asia at least, so you're making all those extra hops to get to the servers. Even just downloading Meteor itself can take 3-4 times longer.